src/Workleap.ComponentModel.DataAnnotations.Tests/ContainsOnlyNonEmptyGuidsAttributeTests.cs

```diff
@@ -40,23 +40,23 @@ public void Validator_TryValidateObject_Returns_The_Expected_Error_Message_When_
         Assert.Equal(expectedErrorMessage, result.ErrorMessage);
     }
 
-    private class ValidData : IEnumerable<object?[]>
+    private sealed class ValidData : IEnumerable<object?[]>
```

Comment:
A better cleanup would be to use a MemberData and a TheoryData<T>

Reply:
Changed to TheoryData